initial content for finaid 2024 (#483)
EuroPython/website: data/pages-content/finaid.md

# Financial Aid

As part of our commitment to the Python community, we are pleased to announce
that we offer special grants for people in need of financial aid to attend
EuroPython.

You’ll find all the information you need to apply for such a grant below.

---

## Financial Aid Sponsors

<div className="finaid-sponsor">
  <a href="https://www.europython-society.org/" target="_blank">
    <img src="/img/eps-logo-white.png" alt="Logo of the EuroPython Society" />
  </a>
</div>

> <!-- The EuroPython Society is sponsoring the Financial Aid Programme with <amount> this year. -->

Reviewer: ```suggestion
The EuroPython Society is sponsoring the Financial Aid Programme with €25,000 this year.
```

> <!-- The EuroPython Society are sponsoring the Financial Aid Programme with <amount> this year. -->

Reviewer 2: ```suggestion
<!-- The EuroPython Society is sponsoring the Financial Aid Programme with <amount> this year. -->
```

The data we collect on this form will exclusively be used by our Financial Aid
team for the selection of grant recipients and the processing of financial
aid refunds. No data is passed on to third parties, except to our accountants
and the bank in order to process the refunds. We use Google Forms for the
processing and Google Drive for storing the relevant information.

For more details on our privacy policies, please see our
[privacy policy][privacy-policy].

## Timeline

Grants will be awarded in two rounds. This makes it easier for us to reallocate
grants in case grant offers were rejected in the first round. Receiving a grant
early is also beneficial for people having to apply for a Schengen visa.

If you’ve applied for the first round, you will automatically be considered
during the second round if you did not receive a grant during the first round.
You do not have to reapply separately for the second round. **In fact, if you
want to increase your chances of receiving a grant, it is recommended to apply
early.**

**Note:** dates in the timeline follow the [Anywhere on Earth][AoE] convention.

<table className="mt-8 finaid-timeline">
<tbody className="text-xl">
  <tr>

>     <td>6 March 2024</td>

Reviewer: ```suggestion
    <td>8 March 2024</td>
```

    <td></td>
    <td>Applications open</td>
  </tr>
  <tr className="font-bold">
    <td>21 April 2024</td>
    <td>
      <img
        src="/img/accents/finaid-round1.svg"
        className="findaid-accent-round-marker"
        alt="round one" />
    </td>
    <td>Deadline for submitting first-round applications</td>
  </tr>
  <tr>

>     <td>19 May 2024</td>

Reviewer: ```suggestion
    <td>8 May 2024</td>
```

    <td>
      <img
        src="/img/accents/finaid-round1.svg"
        className="findaid-accent-round-marker"
        alt="round one" />
    </td>
    <td>First round of grant notifications</td>
  </tr>
  <tr>

>     <td>12 May 2024</td>

Reviewer 2: This should happen _after_ the 19th of May, right?
How do I accept a grant if I haven't been notified I got it?

Reviewer: Should be resolved here https://github.com/EuroPython/website/pull/483#discussion_r1517528292

  conference ticket that gives you access to the three main conference days.

- If you have already purchased a ticket prior to being granted a free Financial Aid ticket voucher, please request a refund for the paid ticket separately before using the voucher to claim your free ticket. The Financial Aid programme cannot reimburse you the cost of tickets you previously paid for directly.

### Travel/Accommodation Grant & Visa Application Fee Grant

If you received a travel/accommodation grant or a visa application fee grant,
you will be reimbursed **after the conference** via PayPal or bank transfer
(SEPA/Swift). We will only reimburse you after we've received your
receipt/invoices and a **photo of your conference badge**.

In addition, please pay close attention to the following:

- The total combined travel/accommodation grant cannot exceed €400, unless otherwise agreed.
- The Visa Application Fee Grant cannot exceed €80.
- Reimbursements will be made **after** the conference via PayPal or bank
  transfer (SEPA and Swift). It’s not possible to be reimbursed before the
  conference or by other means, such as cash.

> - Note that we might experience difficulty transacting with certain  countries

Reviewer 2: ```suggestion
- Note that we might experience difficulty transacting with certain countries
```

  due to sanctions or international regulations.
- Reimbursements will only be made after we’ve received your
  travel/accommodation receipts/invoices. The deadline to submit your
  receipts/invoices is **21 July 2024** [Anywhere on Earth][AoE].
- Please make sure that your receipts/invoices contain the information necessary
  for us to validate them. At the very least, they should contain your name, the
  date, the amount, and specify what the receipt/invoice is for.
- Reimbursements will only be made to individuals who actually attended the
  conference. Please include a **photo of your conference badge** with your
  receipts/invoices.
- Please consider the risk of travel restrictions: choose flexible booking
  options that allow you to cancel in case you can’t actually make it to the
  conference.
- If you’ve applied for a grant or accepted a grant, but cannot make it to the

>   conference, please contact us at [finaid@europython.eu][email-finaid]. This

Reviewer 2: Kind of goes without saying, but explicit is better than implicit ✨ , no?

```suggestion
  conference, please contact us at [finaid@europython.eu][email-finaid] as soon as possible. This
```
